packages/editor/src/components/collab-sidebar/style.scss

```
@@ -135,6 +135,40 @@
 	display: block;
 }
 
+.editor-collab-sidebar-panel__skip-link {
+	// Visually hidden until focused (skip link pattern)
+	position: absolute;
+	top: -9999px;
+	left: -9999px;
+	width: 1px;
+	height: 1px;
+	overflow: hidden;
+	clip-path: inset(50%);
+
+	&:focus {
+		// Restore normal positioning and visibility when focused
+		position: absolute;
+		top: 20px;
+		left: 85px;
+		width: auto;
+		height: auto;
+		overflow: visible;
+		clip-path: none;
+		z-index: 100000;
+		margin-bottom: 0;
+		background-color: $white;
+		border-radius: $radius-small;
+		box-shadow: $elevation-medium;
+	}
+
+	&:focus:hover,
+	&:focus:active {
+		// Maintain solid background on all interactions
+		background-color: $white !important;
+		opacity: 1 !important;
+	}
+}
```

```suggestion
// Visually hidden until focused (skip link pattern)
.editor-collab-sidebar-panel__skip-link {
	position: absolute;
	top: -9999px;
	right: -9999px;
	overflow: hidden;
	clip-path: inset(50%);
	background: $white !important;

	&:focus {
		top: $grid-unit-10;
		right: $grid-unit-10;
		overflow: visible;
		clip-path: none;
	}
}
```
It could be simpler.

---

@t-hamano - Updated all the changes. Here is the updated video.


https://github.com/user-attachments/assets/49839cb0-de4a-44d2-be4a-6d4b47fcee1d